src/scenes/ModulesRouter/ModuleFieldDisplay.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+import { Button } from '@terraware/web-components';
+
+import strings from 'src/strings';
+
+const ModuleFieldDisplay = ({
```

Comment:
I think we can name this to be ModuleEventCard or something like that, and make the props take in a dateTime specifically.

Or if you are imagining this will be used in other places, we can move it to components/

Seems like this pattern is only used for events according to Figma.